server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+[id="proc-creating-oidc-client_{context}"]
+==== Creating an Open ID Connect Client
+
+. Click the `Clients` left menu item.  
+. Click the `Create` button. This will bring you to the `Add Client` page.
```

Comment:
I think you need to keep this sentence "This will bring you to the `Add Client` page." after the client.png graphic.  It introduces the second graphic.

Reply:
Do we need both images? Please take a closer look at using too many images. Also, if you must use it, change to:
"Click 'Create' to access the 'Add Client' page."

Reply:
tbh, I don't think there is any point in keeping the images. I do not think that they bring anything special to the text and we do not use images much within the rest of the doc. I am taking them out.